ui/package.json

```diff
@@ -162,7 +162,7 @@
     "jasmine-core": "~3.3.0",
     "jasmine-spec-reporter": "~4.2.1",
     "jest": "^26.6.3",
-    "karma": "^5.0.0",
```

Comment:
All v5 karma versions depend on vulnerable ua-parser-js versions, so we need to upgrade to 6.  6.3.2 is the latest version.